src/core/deprecated/shaderlib/project/project-deprecated.glsl.js

```diff
@@ -21,6 +21,7 @@
 export default `\
 // Backwards compatibility
 uniform mat4 modelMatrix;
+uniform mat4 viewMatrix;
```

Comment:
this looks like a deprecated file, do we need to change this?

Reply:
It is still available although the name has "deprecated" and this is temp fix, will implement a better solution later on as I proposed in my RFC, right now need focus on the packed-point-cloud rendering project, and make it unblocked.